libs/fills/src/lib/fills-table.tsx

```diff
@@ -315,7 +314,16 @@ export const getRoleAndFees = ({
   } else {
     return { role: '-', fees: undefined };
   }
-  return { role, fees };
+
+  // We make the assumption that the market state is active if the maker fee is zero on both sides
```

Comment:
This is a very helpful comment, is it true that its when the market is active? I thought it was more due to trading mode?

Reply:
The market is active only if the trading mode is "trading continuously"